src/httpClient.ts

```diff
@@ -93,6 +94,12 @@ export class HttpClient {
       },
     });
 
+    if (resp.status === UNAUTHORIZED) {
+      throw new Error(
+        `Unauthorized. Please check that your HAMMING API key is correct by visiting: https://app.hamming.ai/settings`,
```

Comment:
`HAMMING_API_KEY` instead of `HAMMING API key`, clearer call to action for what to check first.

Not sure if any user is pointing to a different environment using `HAMMING_BASE_URL`. If so, remove the URL part of the message if that value has been set, it will be confusing.

Reply:
Addressed!